dashboard/sections/http/main.tf

```diff
@@ -21,7 +22,42 @@ module "incoming_latency" {
   filter = concat(var.filter, ["metric.type=\"run.googleapis.com/request_latencies\""])
 }
 
+locals {
+  // gmp_filter is a subset of var.filter that does not include the "resource.type" string
+  gmp_filter = [for f in var.filter : f if !strcontains(f, "resource.type")]
+}
+
+output "gmp_filter" {
+  value = local.gmp_filter
+}
+
+output "var_filter" {
```

Comment:
We don't need this, since anything that wants to share its value can already just get `.filter`

Reply:
Done